.github/workflows/rad_update_bot.yml

```diff
@@ -19,6 +19,7 @@ jobs:
       POSTGRES_DB: rad_common_development
       POSTGRES_USER: postgres
       POSTGRES_PASSWORD: postgres
+      CI: true
```

Comment:
using the same pattern and seems like a good name for this environment variable since the bot is arguably part of our continuous integration machinery